ci_framework/roles/build_containers/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install required packages
+  become: true
+  ansible.builtin.package:
+    name:
+      - python3-devel
+      - python3-pip
+      - podman
+      - buildah
+    state: latest
+
+- name: Check if directory exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_build_containers_tcib_dir }}"
+  register: tcib_dir
+
+- name: Get tcib from repository
+  ansible.builtin.git:
+    accept_hostkey: true
+    dest: "{{ cifmw_build_containers_tcib_dir}}/tcib"
+    repo: "{{ cifmw_build_containers_tcib_src }}"
+  when: not tcib_dir.stat.exists | bool
+
+  # FIXME(Chandan/arxcruz): Skip usage of sudo installation
+- name: Install tcib package
+  become: true
+  ansible.builtin.shell: |
```

Comment:
indeed, sudo shouldn't be required. In the cifmw case, we also have a virtual env at some point - via the repo_setup role. You may want to re-use it. You can check the role as to how we're bootstrapping and using it?